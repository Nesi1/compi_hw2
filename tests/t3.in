void main()
{
    int x = 4+5*8;
}