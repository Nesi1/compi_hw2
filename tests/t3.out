6: RetType ->  VOID
7: Formals -> epsilon
31: Type -> INT
38: Exp -> NUM
38: Exp -> NUM
38: Exp -> NUM
35: Exp -> Exp BINOP Exp
35: Exp -> Exp BINOP Exp
16: Statement -> Type ID ASSIGN Exp SC
12: Statements -> Statement
4: FuncDecl -> RetType ID LPAREN Formals RPAREN LBRACE Statements RBRACE
2: Funcs -> epsilon
3: Funcs -> FuncDecl Funcs
1: Program -> Funcs
